src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradeForm/Controllers/ProPlanController/PriceCallout/PriceCallout.tsx

```diff
@@ -20,7 +19,7 @@ import Icon from 'ui/Icon'
 import { UpgradeFormFields } from '../../../UpgradeForm'
 
 interface PriceCalloutProps {
-  newPlan?: PlanName
+  newPlan?: Plan
```

Comment:
NewPlan is now **ALWAYS** a Plan object and never just the plan "name"... it's more inline with what the variable says anyway lol

Reply:
Update: This is actually IndividualPlanSchema rather than Plan to better reflect the actual type